docs/resources/trigger.md

```diff
@@ -110,40 +110,40 @@ resource "wallarm_trigger" "attack_trigger" {
 
 ## Argument Reference
 
-* `client_id` - (Optional) ID of the client to apply the trigger to. The value is required for multi-tenant scenarios.
-* `template_id` - (Required) Trigger condition. A condition is a system event to be notified about. Can be:
+* `client_id` - (optional) ID of the client to apply the trigger to. The value is required for [multi-tenant scenarios][2].
+* `template_id` - (**required**) trigger condition. A condition is a system event to be notified about. Can be:
   - `user_created` for a user added to the company account in Wallarm Console
   - `attacks_exceeded` for detected attacks number exceeded the specified value
   - `hits_exceeded` for detected hits number exceeded the specified value
   - `incidents_exceeded` for detected incidents number exceeded the specified value
   - `vector_attack` for detected attack vectors number exceeded the specified value
```

Comment:
vector is incorrect term

Reply:
Fixed in desriptive text (attack vectors > malicious payloads), but not in the value name. @aoleynikov-wallarm, are you planning to rename the value?

Reply:
It must be changed in the API first. Terraform provider is just a tiny wrapper over the API, so whatever resources exist in the API, with whatever attributes, they're the same here.

https://developer.hashicorp.com/terraform/plugin/hashicorp-provider-design-principles#resource-and-attribute-schema-should-closely-match-the-underlying-api